File: TextSearchNet/model/IdStatus.cs
using Newtonsoft.Json;

namespace com.hideakin.textsearch.model
{
    public class IdStatus
    {
        [JsonProperty("uid")]
        public int Uid { get; set; }

        [JsonProperty("gid")]
        public int Gid { get; set; }

        [JsonProperty("fid")]
        public int Fid { get; set; }

        public override string ToString()
        {
            return string.Format("UID.next={0} GID.next={1} FID.next={2}", Uid, Gid, Fid);
        }
    }
}

File: TextIndexApi/src/main/java/com/hideakin/textsearch/index/model/IdStatus.java
package com.hideakin.textsearch.index.model;

public class IdStatus {

	private int uid;

	private int gid;
	
	private int fid;
	
	public IdStatus() {
		this(-1, -1, -1);
	}

	public IdStatus(int uid, int gid, int fid) {
		this.uid = uid;
		this.gid = gid;
		this.fid = fid;
	}

	public int getUid() {
		return uid;
	}

	public void setUid(int uid) {
		this.uid = uid;
	}

	public int getGid() {
		return gid;
	}

	public void setGid(int gid) {
		this.gid = gid;
	}

	public int getFid() {
		return fid;
	}

	public void setFid(int fid) {
		this.fid = fid;
	}

}
